src/lib/layer-manager.js

```diff
@@ -198,6 +207,22 @@ export default class LayerManager {
 
   // PRIVATE METHODS
 
+  _getPickingBuffer() {
+    const {gl} = this.context;
+
+    // Set up a frame buffer if needed
+    if (this.context.pickingFBO === null ||
+      gl.canvas.width !== this.context.pickingFBO.width ||
```

Comment:
luma.gl v4 has `framebuffer.resize(...)` that can be called every frame, it only resizes attachment when needed.

`FramebufferObject` is deprecated and just an alias for `Framebuffer`